src/views/Login.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+    <div class="grid grid-cols-2 login">
+        <div class="left_content">
+            <div class="logo-area">
+                <img class="h-14 mt-16 ml-14" src="../assets/views/securious-logo.svg" alt="Securious"/>
+            </div>
+            <div class="login-form mt-24 max-w-sm flex flex-col ml-40">
+                <div class="form-title">Einloggen</div>
+                <Form class="flex flex-col space-y-3 mt-4" @submit="login">
+                    <Field class="inp rounded-none p-2 text-xs" name="email" type="email" placeholder="Ihre@email.com" required />
+                    <Field class="inp rounded-none p-2 text-xs" name="password" type="password" placeholder="Passwort" />
+                    <div class="link-text text-right text-sm">Passwort vergessen?</div>
+                    <div class="grid grid-cols-3">
+                        <button class="col-start-2 col-end-2 rounded-full text-white p-2">LOGIN</button>
+                    </div>
+                </Form>
+                <div class="register flex justify-center space-x-4 pt-4">
+                    <div class="text-xs">
+                        Noch kein Account? 
+                    </div>
+                    <div class="text-xs link-text"> 
+                        <router-link to="/">Registrieren</router-link>
+                    </div>
+                </div>
+            </div>
+            <div class="footer-text text-sm text-center m-12">
+                Falls Sie noch das alte passwortlose System von securious benutzt haben und kein Password haben, <router-link class="link-text" to="/">klicken Sie hier</router-link> Sie hier um eines zu erstellen. 
+            </div>
+            <div class="footer-link link-text text-sm mt-20 flex justify-center space-x-4">
+                <div>Impressum</div>
+                <div>Datenchutz</div>
+            </div>
+        </div>
+        <div class="right_content p-3">
+            <img src="../assets/views/login-page.svg" alt="Securious login page image"/>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+    import { Field, Form } from 'vee-validate'
+    import store from '../store'
+    import router from '../router'
+
+    export default{
+        components: {
+            Field,
+            Form,
+        },
+
+        methods: {
+            login(values) {
+                store.dispatch({
+                    type: 'LOGIN',
+                    values
+                });
+                router.push('/dashboard')
+            }
+        },
+    };
+</script>
+
+<style scoped>
+    @media (max-width: 576px) {
```

Comment:
Possibly use Tailwind's class if it exists? Also, if this is used comment, we can add this to style.css.